static/include/app/apis/generated/data.md

```diff
@@ -135,16 +159,18 @@ You can also find your binary data under the **Images**, **Point clouds**, or **
 from viam.utils import create_filter
 from viam.proto.app.data import Filter, TagsFilter, TagsFilterType
 
-
 # Get data captured from camera components
-my_data = []
-last = None
-my_filter = create_filter(component_name="camera")
-while True:
-    data, count, last = await data_client.binary_data_by_filter(my_filter, last)
-    if not data:
-        break
-    my_data.extend(data)
+camera_filter = create_filter(
+    component_name="camera-1"
+    )
+    
+data, count, last = await data_client.binary_data_by_filter(
+    filter=camera_filter, 
+    limit=1, 
+    include_binary_data=True
+)
+for binary in data:
+    print(binary)
 
 # Get untagged data from a dataset
```

Comment:
@npentrel I could only get the above method to work. Did you test this? I assumed you did and it works so I wouldn't mind keeping both methods

Reply:
How were you testing it and what issue did you see?

Reply:
Put the code sample in a script then off the command line. I don't remember well but I think it was just hanging. I can retest with your code sample

Reply:
My best guess is you didn't have any unlabeled data in the dataset you were testing this with.

Reply:
working now, readded